README.md

```diff
@@ -11,6 +11,29 @@ The main goal of this project is to provide a compatible, reliable and efficient
 
 The project is in early development.
 
+### Installation
+`OpenCombine` contains of two targets: `OpenCombine` and `OpenCombineDispatch`.
```

Comment:
Actually there are 3 targets, one of them is just not really "public". I think it's worth mentioning that:

>`OpenCombine` contains two public targets: `OpenCombine` and `OpenCombineDispatch` (the third one, `COpenCombineHelpers`, is considered private, don't import it in your projects).

Also, it would be good to mention what `OpenCombineDispatch` is, as [this can be non-obvious](https://github.com/broadwaylamb/OpenCombine/issues/94#issuecomment-546593378). Like so:

>OpenCombine itself doesn't depend on anything but the platform libraries. Not even Foundation or Dispatch. For using OpenCombine with Dispatch (for example for using `DispatchQueue` as `Scheduler` for operators like `debounce`, `receive(on:)` etc.) you need to import both `OpenCombine` and `OpenCombineDispatch`.

The latter note should probably go to the "Usage" section, which needs to be created first.

Reply:
Good points, I'll add them. 👍